integration/jnifuse/native/src/main/native/libjnifuse/operation.cc

```diff
@@ -177,12 +177,14 @@ FlushOperation::FlushOperation(JniFuseFileSystem *fs) {
 }
 
 int FlushOperation::call(const char *path, struct fuse_file_info *fi) {
+  LOGD("into flush: %s", path);
```

Comment:
is it needed? in my other PRs, i am removing all the LOGD in those XXOperation.call commands.
This is because when alluxio.fuse.debug.enabled=true, all the libfuse calls will be logged in fuse.log and fuse.out.
No need to add libjnifuse DEBUG log

Reply:
logs are removed